packages/editor/CodeMirror/extensions/rendering/renderBlockImages.ts

```diff
@@ -72,8 +112,12 @@ class ImageWidget extends WidgetType {
 		return container;
 	}
 
+	private get cacheKey() {
+		return `${this.src_}_${this.width_ ?? ''}`;
```

Comment:
```suggestion
		return `${this.src_}_${this.width_ ?? ''}_${this.reloadCounter_}`;
```

- Consider including `this.reloadCounter_` in the `cacheKey`. Doing so should refresh the height when image changes in an external editor or is changed due to sync.

Thank you for working on this!

Reply:
I've made the suggested change, testing done:

- Opened images with external editor (triggering file watch) and resized them. Verified new size is immediately reflected in the editor and then scrolled document and verified there was no unusual scroll behavior
- Sync tests - modify note images in Client B (resize HTML image via syntax, open in external editor and resize, delete images and add new images), wait for sync to propagate to Client A and then scrolled document
- Same resource ID in different notes with different syntax (markdown vs html, html with different dimensions)

NOTE: I noticed an issue where if you're looking at a note, and sync updates the note to include a *new* image resource, that image won't be rendered (shown as a broken image) until you scroll away from the image and then back. However, this also happens in the current version of Joplin (might be related to the syntax tree iteration in makeBlockReplaceExtension)

There's also another pre-existing issue where when you access a note with many images, some of them may not render until you left click somewhere in the note which might be related to the above, I created an issue for this: https://github.com/laurent22/joplin/issues/13963